### PR TITLE
Task101/dump frequencies gvf vcf

### PR DESCRIPTION
**Pull request by at7**
Generate GVF and VCF dumps with 1000 Genomes frequencies. I will provide test data and a script for testing. I also still need to write the documentation page in confluence. The PR is against master.

### CODE REVIEW

File: modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/DumpFrequencies.pm [hunk 1]
```diff
@@ -0,0 +1,274 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
```
Comment (dglemos) [inline modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/DumpFrequencies.pm]: Copyright

File: modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/FinishDumpFrequencies.pm [hunk 2]
```diff
@@ -0,0 +1,195 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
```
Comment (dglemos) [inline modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/FinishDumpFrequencies.pm]: Copyright

File: modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/ReleaseFrequencyDumps_conf.pm [hunk 3]
```diff
@@ -0,0 +1,203 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
```
Comment (dglemos) [inline modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/ReleaseFrequencyDumps_conf.pm]: Here too.

**Comment by dglemos** [comment]
Thanks for adding all the comments. I'm happy to merge it.

**Comment by at7** [comment]
Thank you for your feedback!

**Pull request merged**